src/utils/typedefs-wrapper.cpp

```diff
@@ -44,10 +44,19 @@ std::size_t convstr(char16_t const* inp, char* outp, int isz, int osz) {
 
 Il2CppString* alloc_str(std::string_view str) {
     il2cpp_functions::Init();
+
+    if (str.data() == nullptr) {
+        return il2cpp_functions::string_new_len("\0", 0);
```

Comment:
No need to use `\0` here, just an empty string already has an implicit `\0` at the end of it, so just use: `""`